content/en/observability_pipelines/production_deployment_overview/aggregator_architecture.md

```diff
@@ -0,0 +1,414 @@
+
+To achieve high durability:
+
+1. Deploy at least two Observability Pipelines Worker instances in each Availability Zone.
+2. Deploy Observability Pipelines Worker in at least two Availability Zones.
+3. Front your Observability Pipelines Worker instances with a load balancer that balances traffic across Observability Pipelines Worker instances. See the horizontal scaling section for more information.
+
+### Mitigating failure scenarios
+
+#### Handling Observability Pipelines Worker process issues
+
+To mitigate a system process issue, distribute Observability Pipelines Worker across multiple nodes and front them with a network load balancer that can redirect traffic to another Observability Pipelines Worker instance as needed. In addition, platform-level automated self-healing should eventually restart the process or replace the node.
+
+#### Mitigating node failures
+
+To mitigate node issues, distribute the Observability Pipelines Worker across multiple nodes and front them with a network load balancer that can redirect traffic to another Observability Pipelines Worker node. In addition, platform-level automated self-healing should eventually replace the node.
+
+#### Handling availability zone failures
+
+To mitigate issues with availability zones, deploy the Observability Pipelines Worker across multiple availability zones.
+
+#### Mitigating region failures
+
+Observability Pipelines Worker is designed to route internal observability data, and it should not failover to another region. Instead, Observability Pipelines Worker should be deployed in all of your regions as recommended in the [network boundaries](#network-boundaries) section. Therefore, if your entire network or region fails, Observability Pipelines Worker would fail with it.
+
+## Disaster recovery
+
+### Internal disaster recovery
+
+Observability Pipelines Worker is an infrastructure-level tool designed to route internal observability data. It implements a shared-nothing architecture and does not manage state that should be replicated or transferred to a disaster recovery (DR) site. Therefore, if your entire region fails, Observability Pipelines Worker would fail with it. Therefore, you should install the Observability Pipelines Worker in your DR site as part of your broader DR plan.
+
+### External disaster recovery
+
+If you're using a managed destination, such as Datadog, Observability Pipelines Worker can facilitate automatic routing data to your Datadog DR site using Observability Pipelines Worker's circuit breaker feature.
+
+## Advanced configurations
+
+### Multiple aggregator deployments
+
+As covered in [network boundaries](#network-boundaries), Datadog recommends to start with one Observability Pipelines Worker aggregator per region. This is to prevent overcomplicating your initial deployment of Observability Pipelines Worker, but there are circumstances where starting with multiple deployments makes sense:
```

Comment:
```suggestion
As covered in [network boundaries](#network-boundaries), Datadog recommends to start with one Observability Pipelines Worker aggregator per region. This is to prevent overcomplicating your initial deployment of Observability Pipelines Worker, but there are circumstances where starting with multiple deployments is ideal:
```